src/ia-menu-slider.js

```diff
@@ -92,31 +92,41 @@ export class IAMenuSlider extends LitElement {
     const actionSection = actionButton
       ? html`<div class="custom-action">${actionButton}</div>`
       : nothing;
+    const headerClass = actionButton ? 'with-secondary-action' : '';
 
     return html`
-      <header>
+      <header class="${headerClass}">
         <div class="details">
           <h3>${label}</h3>
           <span class="extra-details">${menuDetails}</span>
         </div>
         ${actionSection}
-        <button class="close" aria-label="Close this menu" @click=${this.closeMenu}>
-          <ia-icon-collapse-sidebar></ia-icon-collapse-sidebar>
-        </button>
+        ${this.closeButton}
       </header>
     `;
   }
 
+  get closeButton() {
+    return html`
+      <button class="close" aria-label="Close this menu" @click=${this.closeMenu}>
+        <ia-icon-collapse-sidebar></ia-icon-collapse-sidebar>
+      </button>
+    `;
+  }
+
   /** @inheritdoc */
   render() {
     return html`
       <div class="menu ${this.sliderDetailsClass}">
-        <ul>
+        ${this.closeButton}
```

Comment:
Does the close button get displayed twice? (here and line 104 above)

Reply:
yes, we found this lil guy missing from initial implementation of the mock.
here's what it looks like now. we can revisit when we get more feedback from internal beta..

![ezgif com-video-to-gif-1](https://user-images.githubusercontent.com/7840857/96172100-bc50bf80-0eda-11eb-9b28-7b84a67a8c4e.gif)